ui/src/components/groups/GroupTable.vue

```diff
@@ -75,6 +75,12 @@
         >
           {{ props.item.members.filter(ev => ev.active).length }}
         </td>
+        <td
+          class="hover-hand"
+          v-on:click="$router.push({ path: '/group-types/' + props.item.id })"
```

Comment:
Fine for now, but we should probably use `name` in the future... (e.g. `push({ name: 'group-type-route-name' })')